### PR TITLE
load lateral yards from nflverse releases and join player infos via `nflreadr::load_players()`

### LINKED ISSUE #356: The multiple laterals file should be hosted in a nflverse repo
https://github.com/nflverse/nflfastR/blob/0fa754dda504b133e03885f6ef53bf8e21a56276/R/aggregate_game_stats.R#L100

Any suggestions where to host it @tanho63? It needs to be updated daily through GHA.

### PR DESCRIPTION
**Pull request by mrcaseb**
closes #356

### CODE REVIEW

**Review by tanho63 — CHANGES_REQUESTED** [review]
Can you also look at https://github.com/nflverse/nflfastR/blob/cb1e06379f94c9cfba6def0f331decbcd42abc48/R/aggregate_game_stats.R#L161 in this PR? 

And/or maybe should look at #317 ?

**Review by tanho63 — DISMISSED** [review]

**Comment by mrcaseb** [comment]
RACR IDs were lost after roster repo updates. We compute them through player info now

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# nflfastR (development version)
+
+* Internal change to `calculate_player_stats()` that reflects new nflverse data infrastructure.
+* `calculate_player_stats()` now joins player information like position or headshot url via `nflreadr::load_players()`.
```
Comment (tanho63) [inline NEWS.md:4]: Can you add a list of new columns here as a reference? (Makes easier to point people at when their code breaks for dupl colnames)

**Review by tanho63 — APPROVED** [review]

**Comment by tanho63** [comment]
lgtm thank you. adding to-do to have news + data dictionary in nflreadr

**Pull request merged**